Directory.Build.props

```diff
@@ -22,6 +22,7 @@ Some common use cases include:
         <PackageIcon>fantomas_logo.png</PackageIcon>
         <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
         <PackageReadmeFile>README.md</PackageReadmeFile>
+        <Tailcalls>true</Tailcalls>
```

Comment:
Could you add it to `Fantomas.fsproj` only? It is really only there that we have code that needs this.